src/CommandParser.ts

```diff
@@ -75,6 +79,15 @@ export class CommandParser {
     return command
   }
 
+  /*
+    Specific function to parse Generate Command data.
+  */
+    public static parseGenerateCommand(args: string[]): IImpersonateCommand{
+      const generateMessage = args.join(" ")
```

Comment:
Should we join the rest of the command in a single string or get just the type using `args[0]`

Reply:
done